docs/stories/04-components/LinkButton.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import { LinkButton } from '@strapi/design-system';
+
+import * as LinkButtonStories from './LinkButton.stories';
+
+<Meta of={LinkButtonStories} />
+
+# LinkButton
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Usage with other routing libraries](#usage-with-other-routing-libraries)
+- [Props](#props)
+
+## Overview
+
+A useful wrapper around the [`Button`](../?path=/docs/components-button--docs) component that allows users to navigate
+to a different location. By default, the `tag` prop is the [`BaseLink`](../?path=/docs/primitives-baselink--docs)
+component, but can be changed to integrate with routing libraries.
+
+[View source](https://github.com/strapi/design-system/tree/main/packages/design-system/src/LinkButton)
+
+## Usage
+
+```js
+import { LinkButton } from '@strapi/design-system';
+```
+
+<Canvas of={LinkButtonStories.Base} />
+
+## Usage with other routing libraries
+
+### React Router
+
+To use the LinkButton component with a routing library (e.g. react-router-dom), you'll need to pass the `NavLink`
+component to the `tag` prop in order to replace the default HTML anchor `<a />`. You'll now be able to pass all
+`NavLink` props.
+
+```jsx
+import { LinkButton } from '@strapi/design-system';
+import { NavLink } from 'react-router-dom';
+
+<LinkButton tag={NavLink} to="/home">
+  Home
+</LinkButton>;
+```
+
+### NextJS
+
+For NextJS, you'll need to wrap the `LinkButton` with the `NextLink` component
```

Comment:
I wonder why that is? IIRC next no longer requires you to put an anchor inside NextLink

Reply:
The change happened [between Next 12 and 13](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#link-component). Since the latest is 15 I think we can put the modern version. So based on their [current docs](https://nextjs.org/docs/pages/api-reference/components/link) I think this should work:

```jsx
import { LinkButton } from '@strapi/design-system';
import NextLink from 'next/link';


<LinkButton tag={NextLink} href="/home">Home</LinkButton>
```

Reply:
well that's even easier ⭐